modules/dnn/src/layers/prior_box_layer.cpp

```diff
@@ -487,9 +487,7 @@ class PriorBoxLayerImpl CV_FINAL : public PriorBoxLayer
         if (_explicitSizes)
         {
             InferenceEngine::Builder::PriorBoxClusteredLayer ieLayer(name);
-
-            CV_Assert(_stepX == _stepY);
-            ieLayer.setStep(_stepX);
+            ieLayer.setSteps({_stepX, _stepY});
```

Comment:
According to it's implementation, we should pass `{_stepY, _stepX}` instead:

https://github.com/opencv/dldt/blob/17e66dc5a6631d630da454506902bd7c25d4170b/inference-engine/src/inference_engine/builders/ie_prior_box_clustered_layer.cpp#L83-L89